crowdsec-docs/unversioned/user_guides/log_centralization.md

```diff
@@ -0,0 +1,268 @@
+
+But because our logs are not in a standard location, we need to configure the acquisition to tell crowdsec where our logs are.
+
+Create a file in `/etc/crowdsec/acquis.d/nginx.yaml` with the following content:
+```
+filenames:
+ - /var/log/remote-logs/nginx/*.log
+labels:
+ type: syslog
+```
+
+Repeat for auth logs, create a file `/etc/crowdsec/acquis.d/ssh.yaml` with the following content:
+```
+filenames:
+ - /var/log/remote-logs/auth/*.log
+labels:
+ type: syslog
+```
+
+Note that we are setting the type label to `syslog`, instructing crowdsec to use the `syslog` parser to extract the actual type from the log itself.
+
+Then, we need to install the nginx collection for crowdsec to be able to detect attacks:
+```bash
+$ sudo cscli collections install crowdsecurity/nginx
+```
+
+Lastly, we will also need to make crowdsec on all interfaces to make sure our web servers can contact LAPI.<br/>
+Edit the file `/etc/crowdsec/config.yaml`, and set `api.server.listen_uri` to `0.0.0.0:8080`:
+```yaml
+api:
+  server:
+    listen_uri: 0.0.0.0:8080
+```
+
+Finally, restart crowdsec:
+```bash
+$ sudo systemctl restart crowdsec
+```
+
+### Remediation components setup
+
+CrowdSec, by itself, will only detect bad behaviors and make decisions about IPs; it will not block them.
+
+To block an IP, you need to install a [remediation component](/unversioned/bouncers/intro.md).
+
+For this guide, we'll be using the [firewall remediation component](/unversioned/bouncers/firewall.mdx) that will add local firewall rules to block malicious IPs.
+
+On your web servers, add the crowdsec repository:
+```bash
+$  curl https://install.crowdsec.net/ | bash
+```
+
+Then, install the firewall remediation component:
+```bash
+$ sudo apt install crowdsec-firewall-bouncer-nftables
+```
+
+We now need to create API keys for both our remediation components.
+On the central server, run the commands:
+```bash
+$ sudo cscli bouncers add fw-bouncer-web-1
+API key for 'fw-bouncer-web-1':
+
+   v3G2V//B4IAEFUkON3zWq5yz331UGZDQlQercn3n5T8
+
+Please keep this key since you will not be able to retrieve it!
+$ sudo cscli bouncers add fw-bouncer-web-2
+API key for 'fw-bouncer-web-2':
+
+   Nda9MreJsUBt/EEmz3TI0Jr6p1a9U2XSx5CEeVfkNRw
+
+Please keep this key since you will not be able to retrieve it!
+```
+
+Now, on each web server, edit the file `/etc/crowdsec/bouncers/crowdsec-firewall-bouncer.yaml` and update the `api_url` options with the IP on the central server, and paste the API key in `api_key`:
+
+```yaml
+api_url: http://<central-server-ip>:8080/
+api_key: <API_KEY>
+```
+
+Finally, restart the remediation component:
+```bash
+$ sudo systemctl restart crowdsec-firewall-bouncer
+```
+
+## Test
+
+Now that everything is setup, it's time to test !
+
+We'll scan one of our web servers, and because both of them are querying the same crowdsec instance if one detects the attack, the other server will also block the attacker.
+
+```bash
+$ nikto -h 52.50.157.217
```

Comment:
Insert a placeholder like `<server_1>` and above

:::info
replace <server_1> with your connected server ip address
:::

cause people will blindly copy and paste on this IP